frontend/dynamic-demo-plugin/src/utils/telemetry.ts

```diff
@@ -0,0 +1,90 @@
+const SEGMENT_KEY = '3Din4v9Wvo5LlHz5omATWCHrrgkliWaF';
```

Comment:
It looks like the SEGMENT_KEY is still defined here?

It would be nice if we can parameterize this somehow during the build so I don't need to change the code, but this is OK for now.

Reply:
Oh, sorry about forgetting remove this! I will remove it now. Thanks for reminding @spadgett .